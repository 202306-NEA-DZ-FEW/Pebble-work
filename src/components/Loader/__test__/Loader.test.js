import { render } from "@testing-library/react";
import React from "react";

import Loader from "../Loader";
describe("Loader", () => {
    it("renders correctly", () => {
        const { asFragment } = render(<Loader />);
        expect(asFragment()).toMatchSnapshot();
    });
});
