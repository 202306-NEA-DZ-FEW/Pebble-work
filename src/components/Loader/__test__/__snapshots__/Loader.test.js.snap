// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<DocumentFragment>
  <div
    class="body"
  >
    <div
      class="wrapper"
    >
      <div
        class="candles"
      >
        <div
          class="lightWave"
        />
        <div
          class="candle1"
        >
          <div
            class="candle1Body"
          >
            <div
              class="candle1Eyes"
            >
              <span
                class="candle1EyesOne"
              />
              <span
                class="candle1EyesTwo"
              />
            </div>
            <div
              class="candle1Mouth"
            />
          </div>
          <div
            class="candle1Stick"
          />
        </div>
        <div
          class="candle2"
        >
          <div
            class="candle2Body"
          >
            <div
              class="candle2Eyes"
            >
              <div
                class="candle2EyesOne"
              />
              <div
                class="candle2EyesTwo"
              />
            </div>
          </div>
          <div
            class="candle2Stick"
          />
        </div>
        <div
          class="candle2Fire"
        />
        <div
          class="sparklesOne"
        />
        <div
          class="sparklesTwo"
        />
        <div
          class="candleSmokeOne"
        />
        <div
          class="candleSmokeTwo"
        />
      </div>
      <div
        class="floor"
      />
    </div>
  </div>
</DocumentFragment>
`;
